packages/math-input/src/components/keypad/__tests__/keypad-v2-mathquill.test.tsx

```diff
@@ -231,4 +247,25 @@ describe("Keypad v2 with MathQuill", () => {
 
         expect(mockMathInputCallback).toHaveBeenLastCalledWith("");
     });
+
+    // CEDAR keypad tests
```

Comment:
**nit**: These aren't CEDAR events.  Let's remove all references to CEDAR in this PR and code.  We map them to CEDAR in webapp, but given this is an open source project that we're working to keep somewhat Khan Academy-agnostic, let's avoid KA-specific terms. :)

Reply:
Okay, sounds good! What would be a better alternative? Just "keyboard event"?

Reply:
Let's use the term "analytics event" ... if you want to refer to a specific event, then I think your idea works great ("keypad event").